fe/src/main/java/org/apache/doris/clone/ColocateTableBalancer.java

```diff
@@ -196,6 +196,11 @@ private synchronized void tryBalanceWhenBackendChange() {
 
         Set<Long> allGroupIds = colocateIndex.getAllGroupIds();
         for (Long groupId : allGroupIds) {
+            if (colocateIndex.isGroupBalancing(groupId)) {
+                LOG.warn("colocate group {} is balancing", groupId);
```

Comment:
Change the log level from 'warn' to 'info'

Reply:
OK